packages/salesforcedx-vscode-core/package.json

```diff
@@ -404,6 +412,10 @@
           "command": "sfdx.force.project.with.manifest.create",
           "when": "!sfdx:internal_dev"
         },
+        {
+          "command": "sfdx.create.manifest",
+          "when": "false"
```

Comment:
why do  we even include this is we're just going to set the when to false?

Reply:
The syntax here is weird, but it's how we prevent the command from appearing in the command palette.

Reply:
sorry dumb follow up q, but if it's not included here at all it'll how up in the command pallet?

Reply:
Correct! I had to check my knowledge on this, but here's the doc that confirms: https://code.visualstudio.com/api/extension-guides/command#controlling-when-a-command-shows-up-in-the-command-palette